test/evergreen.yml

```diff
@@ -97,11 +97,35 @@ functions:
       working_dir: "wiredtiger"
       shell: bash
       script: |
-        # Fetch the gperftools library.
```

Comment:
These changes need to be double checked. I don't think this is a simple backport for evergreen.yml because there are a lot of changes between 6.0 and develop. I think we should leave this deleted section and instead focus on posix_configure_flags over DEFINED_EVERGREEN_CONFIG_FLAGS (DEFINED_EVERGREEN_CONFIG_FLAGS isn't in 6.0). Also DHAVE_BUILTIN_EXTENSION_ZSTD might not be set as aggressively so I would try it first without any of these changes in this section to see if it will work.